src/content/docs/recipes/layout/center-a-layout.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to Center Layouts
+sidebar:
+  order: 2
+  label: Center a Layout
+---
+
+## Problem
+
+You want to center a widget within some area of your TUI's layout.
```

Comment:
How about we rename this file to `center-a-widget` then?

Reply:
https://github.com/ratatui/ratatui-website/commit/ee088076c7743466a5b98187dd61d460382e04c3